modules/cnv-monitoring-upgrade-status.adoc

```diff
@@ -52,4 +52,4 @@ Available          True  Reconcile completed successfully
 Progressing        False Reconcile completed successfully
 Degraded           False Reconcile completed successfully
 Upgradeable        True  Reconcile completed successfully
-----
\ No newline at end of file
+----
```

Comment:
Not sure what git is picking up here. This code block hasn't changed from master